src/debian/stretch/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM debian:stretch
+
+# Dependencies for generic .NET Core builds and the base toolchain we need to 
+# build anything (clang, cmake, make and the like)
+RUN apt-get update \
+    && apt-get install -y \
+            clang \
+            cmake \
+            curl \
+            g++ \
+            gettext \
+            git \
+            gnupg \
+            libcurl4-openssl-dev \
+            libgdiplus \
```

Comment:
What is the libgdiplus needed for?

Reply:
I'm not sure, but it's in most of our Dockerfiles e.g. [Debian8.2](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/src/debian/8.2/coredeps/Dockerfile#L23)